back/engines/commercial/user_custom_fields/spec/models/user_custom_fields/representativeness/categorical_distribution_spec.rb

```diff
@@ -2,17 +2,24 @@
 
 require 'rails_helper'
 
-RSpec.describe UserCustomFields::Representativeness::RefDistribution do
-  subject(:ref_distribution) { build(:ref_distribution) }
+RSpec.describe UserCustomFields::Representativeness::CategoricalDistribution do
+  subject(:ref_distribution) { build(:categorical_distribution) }
 
   describe 'factory' do
     it { is_expected.to be_valid }
   end
 
-  it { is_expected.to belong_to(:custom_field) }
+  # This patch is needed to test counts validations.
```

Comment:
Doing this may impact other tests, because the patch is not removed. Can't you use https://relishapp.com/rspec/rspec-mocks/docs/configuring-responses/block-implementation ?

Reply:
Good catch! I replaced the patch with a helper method.

Reply:
Very good point!